profile/2020-09-01-hybrid.json

```diff
@@ -0,0 +1,625 @@
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/Microsoft.Web/stable/2016-09-01/AppServicePlans.json"
+        },
+        {
+          "resourceType": "serverFarms/metrics",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/Microsoft.Web/stable/2016-09-01/AppServicePlans.json"
+        },
+        {
+          "resourceType": "serverFarms/usages",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/Microsoft.Web/stable/2016-09-01/AppServicePlans.json"
+        }
+      ],
+      "2016-03-01": [
+        {
+          "resourceType": "availableStacks",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/Provider.json"
+        },
+        {
+          "resourceType": "deploymentLocations",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/ResourceProvider.json"
+        },
+        {
+          "resourceType": "georegions",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/ResourceProvider.json"
+        },
+        {
+          "resourceType": "listSitesAssignedToHostName",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/ResourceProvider.json"
+        },
+        {
+          "resourceType": "publishingUsers",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/ResourceProvider.json"
+        },
+        {
+          "resourceType": "recommendations",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/Recommendations.json"
+        },
+        {
+          "resourceType": "sourceControls",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/ResourceProvider.json"
+        },
+        {
+          "resourceType": "validate",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/ResourceProvider.json"
+        }
+      ]
+    }
+  },
+  "data-plane": {
+    "microsoft.keyvault": {
+      "2019-09-01": [
+        {
+          "resourceType": "*",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/keyvault/data-plane/Microsoft.KeyVault/stable/2016-10-01/keyvault.json"
```

Comment:
path is not right, either

Reply:
there is no KV swagger for this api version

Reply:
updated to the correct path